tests/integration/modules/mac_service.py

```diff
@@ -70,17 +63,18 @@ def test_launchctl(self):
         '''
         # Expected Functionality
         self.assertTrue(
-            self.run_function('service.launchctl', ['error', 'bootstrap', 64]))
+            self.run_function('service.launchctl',
+                              ['error', 'bootstrap', '64']))
         self.assertEqual(
             self.run_function('service.launchctl',
-                              ['error', 'bootstrap', 64],
+                              ['error', 'bootstrap', '64'],
```

Comment:
cmdmod.py has gracefully handled non-string arguments since the 2016.3 branch, so this is not a bug in cmdmod.py. The problem here is that for some incomprehensible reason all of the mac modules are using a re-implemented version of ``cmd.run_all``.